src/DesktopVirtualization/exports/Remove-AzWvdUserSession.ps1

```diff
@@ -94,7 +94,7 @@ param(
     [Parameter()]
     [Microsoft.Azure.PowerShell.Cmdlets.DesktopVirtualization.Category('Query')]
     [System.Management.Automation.SwitchParameter]
-    # Force flag to login off userSession.
+    # Specify to force userSession deletion.
```

Comment:
The description change in folder export/*.ps1 will be override in next iteration. To update description effectively, please 
1. Raise a PR to generation branch, update this parameter description in directive like below:
```
directive:
  - where:
      parameter-name: Force
      verb: Remove
      subject: {yourSubject}
    set:
      parameter-description: 'Specify to force userSession deletion.'
```